src/clipboard/copy.ts

```diff
@@ -1,10 +1,10 @@
 import {copySelection} from '../document'
-import type {Instance} from '../setup'
-import {writeDataTransferToClipboard} from '../utils'
+import {Instance} from '../setup'
+import {getActiveElement, writeDataTransferToClipboard} from '../utils'
 
 export async function copy(this: Instance) {
   const doc = this.config.document
-  const target = doc.activeElement ?? /* istanbul ignore next */ doc.body
+  const target = getActiveElement(doc) ?? doc.activeElement ?? doc.body
```

Comment:
```suggestion
const target getActiveElementOrBody(doc)
```
We should use the util that already provides this fallback.

Reply:
I had tried this, but it didn't work. I will investigate a bit.

Reply:
All of these should be fixed now.